bdk-ffi/src/wallet.rs

```diff
@@ -154,8 +161,7 @@ impl Wallet {
 
     pub(crate) fn sign(
         &self,
-        psbt: Arc<Psbt>,
-        // sign_options: Option<SignOptions>,
```

Comment:
nit: might be nice to keep this on a separate line, but at the same time not any sort of blocker to approving this PR (and we might just remove the whole commented line before 1.0 beta anyway on bdk-ffi)

Reply:
Done!